### DISCUSSION TITLE
<Suggest platform for video call, audio call and messaging to integrate with android and explain some details.>

### DISCUSSION THREAD

**Discussion by ShineThyuZan** [community/community#62565] [Other Feature Feedback, Questions, & Ideas]
### Select Topic Area

Question

### Body

pls suggest the question

**Comment by Scholar17** [comment]
I want to suggest Agora.io in which platform you can integrate video call, audio call and messaging for mobile. You can get free tier advantages for video call, audio call and messaging for testing. You can use agora rtc full sdk for video call and audio call and you can use agora rtc chat sdk for messaging. Please see more detail in [agora.io](https://www.agora.io/en/).

**Comment by ShineThyuZan** [comment]
Ok i try

**Comment by queenofcorgis** [comment]
GitHub's [Acceptable Use Policies](https://docs.github.com/en/site-policy/acceptable-use-policies/github-acceptable-use-policies) prohibits coordinated or inauthentic activity like rapid questions and answers. As a result, we'll be unmarking the answer and locking this post. Please note any future violations may result in a temporary or indefinite block from the Community. Thanks for understanding.
